packages/pyolite-kernel/src/worker.ts

```diff
@@ -26,6 +26,11 @@ let resolveInputReply: any;
 declare let Comlink: any;
 importScripts('https://unpkg.com/comlink/dist/umd/comlink.js');
 
+enum Mode {
+  file = 32768,
```

Comment:
it's worth indicating this is a decimal mapping of an octal, e.g. https://github.com/gzuidhof/starboard-python/blob/88327bb56f7e2f673cf1ea4c5ff1e0a68c50e288/src/worker/emscripten-fs.ts#L67